lncfg/db.go

```diff
@@ -144,6 +154,9 @@ func (db *DB) Init(ctx context.Context, dbPath string) error {
 
 	case db.Backend == PostgresBackend:
 		common_sql.Init(db.Postgres.MaxConnections)
+
+	case db.Backend == SqliteBackend:
+		common_sql.Init(0)
```

Comment:
Wouldn't this need more than one if we're keeping the existing namespace splitting (channeldb, macaroons, tower, etc, etc)?